### PR TITLE
implement `gofish search`

### LINKED ISSUE #33: implement `gofish search [food]`
Implement a fuzzy search engine to find available fish food for install.

### PR DESCRIPTION
**Pull request by bacongobbler**
closes #33

### CODE REVIEW

File: cmd/gofish/search.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+					for _, found := range fuzzy.Find(keyword, foodNames) {
+						foundFood[found] = true
+					}
+				}
+			}
+			var foodList []string
+			for food := range foundFood {
+				foodList = append(foodList, food)
+			}
+			fmt.Println(strings.Join(foodList, "\t\t\t"))
```
Comment (notlmn) [inline cmd/gofish/search.go]: `\n` instead of `\t`?

Comment (bacongobbler) [reply cmd/gofish/search.go]: The output mimics the same as `gofish list`: https://github.com/fishworks/gofish/blob/master/cmd/gofish/list.go

Then again, we probably need to reasonably split into a new line every few instances. My terminals all word wrap, but maybe someone else has word wrap disabled.

Comment (notlmn) [reply cmd/gofish/search.go]: My idea is that `\n` makes it possible to list fish name, fish version, and barrel from which it was installed!

Sample [scoop](https://github.com/lukesampson/scoop) output (`app-name app-version [bucket]`):
```
PS C:\Windows\system32> scoop list
Installed apps:

  cpu-z 1.84 [extras]
  losslesscut 1.10.0

PS C:\Windows\system32>
```

Comment (bacongobbler) [reply cmd/gofish/search.go]: That's not a bad idea! I'm off for the weekend but would you mind opening a new ticket with your thoughts? I have some ideas/thoughts around `gofish info` etc that may influence this :+1:

**Pull request merged**